package.json

```diff
@@ -11,12 +11,13 @@
     "icon": "resources/ISGCncEditor.png",
     "license": "SEE LICENSE IN LICENSE",
     "engines": {
-        "vscode": "^1.59.0"
+        "vscode": "^1.73.1"
```

Comment:
Was mich hier wundert. Es gibt unten in der Datei die DevDependencies. Gibt es nicht auch noch andere Dependencies für die Extension oder muss die VS Code Version alles mitbringen?

Reply:
Man kann noch "normale" dependencies definieren. Alles in den dev dependencies sollte eigentlich wirklich nur Entwicklungsnotwendig sein, oder wie siehst du das?
Das einzige bei dem ich ins schwanken geraten bin ist "peggy" was für unseren parser ist. Das wird aber nur für Typisierung genutzt im Code und fällt daher im generierten Javascript code eh weg.

Reply:
Ja ich dachte, wenn die package.json generiert wird, werden auch direkt die Dependencies aufgelistet. Oder hast du die package.json nicht über den Befehl neu erzeugt?

Reply:
Die package.json wird manuell geschrieben. Dependencies werden teilweise auch durch vscode automatisch ergänzt wenn ich sie im Code verwende. 
Aus der package.json wird dann die package-lock.json automatisch generiert